package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "smol-toml",
-	"version": "1.1.4",
+	"version": "1.1.5",
```

Comment:
At the risk of being a pest, I think changing the primary export from ESM to CJS should be considered a breaking change (i.e., semver major increment vs. patch increment).

Reply:
I see why that'd be the case; but the change shouldn't have any effect on downstream users (the package remains compatible with ESM-style imports) so I'm unsure if a semver major is necessary here 🤔

Reply:
You're right! I hadn't realized existing uses would continue to work, sorry.